v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/dlq/KafkaDeadLetterQueueOptions.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.kafka.dlq;
+
+import com.google.cloud.teleport.metadata.TemplateParameter;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.PipelineOptions;
+
+/**
+ * {@link KafkaDeadLetterQueueOptions} is used for any Dead letter queue sinks for the failed
+ * records.
+ */
+public interface KafkaDeadLetterQueueOptions extends PipelineOptions {
+  String KAFKA_DLQ_GROUP_NAME = "Kafka Dead Letter Queue";
```

Comment:
nit: Do you think it makes more sense, if we keep the name as just `Dead Letter Queue`. I am thinking so that wherever this is inherited it automatically goes into the `Dead Letter Queue` group and not a separate `Kafka Dead Letter Queue` group

Reply:
+1, I'd make it "Dead Letter Queue" (so that when we have both Kafka DLQ and BQ DLQ they would go to the same parameter group).